docs/quick-lint-js.schema.config

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Nit: I'd rename this file to `quick-lint-js.config.schema.json`.

* This file is not a config file, so `.config` at the end is not appropriate.
* Several people suggest a `.schema.json` suffix: https://stackoverflow.com/questions/9391370/json-schema-file-extension